cmd/upgrade/operatorroles/cmd.go

```diff
@@ -389,8 +422,9 @@ func buildMissingOperatorRoleCommand(missingRoles map[string]*cmv1.STSOperator,
 			"\t--role-name %s \\\n"+
 			"\t--assume-role-policy-document file://%s \\\n"+
 			"%s"+
-			"\t--tags %s",
-			roleName, filename, permBoundaryFlag, iamTags)
+			"\t--tags %s \\\n"+
+			"\t--path %s",
```

Comment:
I think we might want to concatenate the `--path` only if `path!=""`, to avoid the case where the user doesn't provide a path (optional) so we won't print `rosa upgrade operator-roles ... --path`

Though from AWS's perspective providing an empty path or no path at all will be the same, WDYT?

Reply:
I wrapped with single quotes so there will always be a value for the path - at least empty